client/client_with_secret.go

```diff
@@ -47,6 +47,10 @@ func NewWithSecret(clientID, clientSecret string, scopeOption ScopeOption, opts
 
 	scopeOption(cws)
 
+	if cws.realm != "addons" {
+		cws.baseURL += "/auth/realms"
+	}
```

Comment:
can you put an additional comment here explaining this? E.g `Keycloak requires this path prefix internally on auth.services.bitrise.io domain (for addons this service is available on app.bitrise.io)`?